src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/schema/charts/gauge.test.ts

```diff
@@ -235,14 +240,16 @@ describe('Gauge Schema', () => {
   it('throws on invalid shape type', () => {
     const input = {
       ...baseGaugeConfig,
+      // @ts-expect-error
       metric: {
         operation: 'count' as const,
         field: 'test_field',
       },
       shape: {
-        type: 'invalid' as const,
+        // @ts-expect-error
+        type: 'invalid',
       },
-    };
+    } satisfies GaugeInput;
```

Comment:
I think it's also good to type the invalid cases then explicitly use `@ts-expect-error` where the type is deliberately invalid.